API/AddContact.php

```diff
@@ -0,0 +1,27 @@
+<?php
+	// include database connection file
+	include "dbConfig.php";
+	include "returnFunctions.php";
+
+	$inData = getRequestInfo();
+	
+	$stmt = $conn->prepare("SELECT * FROM Contacts WHERE FirstName=?, LastName=?, Email=?, Phone=?, UserID=?");
+	$stmt->bind_param("ssssi", $inData["FirstName"], $inData["LastName"], $inData["Email"], $inData["Phone"], $inData["UserID"]);
+	$stmt->execute();
+
+	$result = $stmt->get_result();
+
+	if ($result->num_rows > 0){
```

Comment:
```suggestion
	if ($stmt->affected_rows > 0){
```

Reply:
Line 14 is checking if there already exits that contact so it needs to check if num rows returned from the search is > 0, the else statement below it inits the stmt to be an insert